resources/definitions/ultimaker.def.json

```diff
@@ -213,6 +213,57 @@
         },
         "jerk_skirt_brim": {
             "minimum_value_warning": 20
+        },
+        "support_pattern": {
+            "value": "'zigzag'"
+        },
+        "support_wall_count": {
+            "value": "1 if support_structure == 'tree' else 0"
+        },
+        "zig_zaggify_support": {
+            "value": true
+        },
+        "support_connect_zigzags": {
+            "value": true
+        },
+        "support_infill_rate": {
+            "value": "80 if support_interface_enable else 15"
+        },
+        "gradual_support_infill_steps": {
+            "value": "2 if support_interface_enable else 0"
+        },
+        "gradual_support_infill_step_height": {
+            "value": "4*layer_height"
+        },
+        "support_interface_height": {
+            "value": "2*layer_height"
+        },
+        "support_offset": {
+            "value": "2*line_width if support_interface_enable else 0"
+        },
+        "support_xy_distance": {
+            "value": "1"
+        },
+        "support_xy_distance_overhang": {
+            "value": "wall_line_width_0"
+        },
+        "minimum_support_area": {
+            "value": "pow((2 + support_offset), 2)"
+        },
+        "support_interface_skip_height": {
+            "value": "layer_height"
```

Comment:
Default value in the fdmprinter file is 0.3mm. However it get overwritten 9 times in the underlying variant files for the BB cores with support_interface_skip_height = layer_height.  So for cleanup I made this the default in ultimaker.def.json. It will deliver lower values for other print modes with could cost a little extra slice time.